indexer/bag_indexer/src/main/scala/uk/ac/wellcome/platform/archive/indexer/bags/BagsIndexConfig.scala

```diff
@@ -1,14 +1,40 @@
 package uk.ac.wellcome.platform.archive.indexer.bags
 
-import com.sksamuel.elastic4s.ElasticDsl.{dateField, keywordField}
+import com.sksamuel.elastic4s.ElasticDsl._
 import com.sksamuel.elastic4s.requests.mappings.FieldDefinition
 import uk.ac.wellcome.platform.archive.indexer.elasticsearch.IndexConfig
 
 object BagsIndexConfig extends IndexConfig {
+  private val suffixTallyFields: Seq[FieldDefinition] =
+    Seq(
+      keywordField("suffix"),
+      intField("count")
+    )
+
+  private val fileFields: Seq[FieldDefinition] =
+    Seq(
+      keywordField("path"),
+      keywordField("name"),
+      keywordField("name"),
+      longField("size"),
+      keywordField("checksum"),
+      keywordField("type")
```

Comment:
Were you going to add a `checksumType` field? (And if so, rename this one to `checksumValue`?)

Reply:
No, I started to do that, but realised it was more effort than I wanted to put in. This is just the "type" / ontologyType thing.